docs/modules/plugins/pages/plugin-web-app.adoc

```diff
@@ -1250,6 +1250,63 @@ When I wait until scroll is finished
 Then page is scrolled to element located by `id(footerElement)`
 ----
 
+==== Script Steps
+
+WARNING: These steps are deprecated and will be removed in VIVIDUS 0.7.0.
+
+The Script Steps verify the presence of JavaScript files in the HTML source code of an active page.
+
+=== Check for JavaScript code in HTML source code with a specific file name (@src attribute)
+
+This step verifies that JavaScript (an element with the <script> tag) with specific file name (@src attribute) is present on the page.
```

Comment:
Please follow the same approach as for other steps: start with a verb in 3rd form in present simple tense